.github/workflows/eslint.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  eslint:
+    name: Run eslint scanning
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+          # We must fetch at least the immediate parents so that if this is
+          # a pull request then we can checkout the head.
+        with:
+          fetch-depth: 2
+
+      # If this run was triggered by a pull request event, then checkout
+      # the head of the pull request instead of the merge commit.
+      - run: git checkout HEAD^2
+        if: ${{ github.event_name == 'pull_request' }}
+
+      - name: Install ESLint
+        run: |
+          npm install eslint@8.57.0
+          npm install utf8
+          npm install jschardet
+
+      - name: Run ESLint
+        run: npx eslint --ext ts extension test tooling --format ./eslint-sarif-formatter.js --output-file eslint-results.sarif
```

Comment:
Here is an example of `test_eslint_ci` action which reuses our `test_eslint` action but uses additional params `--format` and `--output-file`, let's use similar one here -  https://github.com/FlowCrypt/flowcrypt-browser/blob/764daf24cbd7b855ef06888305dd76f45483f7a0/package.json#L80

Reply:
I understand completely. Will do.